### PR TITLE
Added private endpoint limitation

### PR DESCRIPTION
**Pull request by Peter-Msft**

### CODE REVIEW

**Comment by Jak-MS** [comment]
@scoriani 
- Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by scoriani — CHANGES_REQUESTED** [review]
The sentence is not accurate, you can use Private Links for the database that contains the External Data Source definition, while you can't use Private Links for those databases that are targets of External Data Sources. Basically, you can use Private Links for the database you're connecting to from your application, but not for those hosting the external tables.

**Review by scoriani — APPROVED** [review]

**Comment by scoriani** [comment]
#sign-off

File: azure-sql/database/elastic-query-overview.md [hunk 1]
```diff
@@ -141,6 +141,7 @@ Elastic query is included in the cost of Azure SQL Database. Note that topologie
 * Column statistics over external tables are currently not supported. Table statistics are supported, but need to be created manually.
 * Cursors are not supported for external tables in Azure SQL Database.
 * Elastic query works with Azure SQL Database only. You cannot use it for querying a SQL Server instance.
+* Private Links are currently not supported with Elastic Query for those databases that are targets of External Data Sources.
```
Comment (Jak-MS) [inline azure-sql/database/elastic-query-overview.md:144]:
```suggestion
* Private links are currently not supported with elastic query for those databases that are targets of external data sources.
```

**Pull request merged**